scripts/go-test.in

```diff
@@ -21,17 +28,50 @@ for arg in "$@" ; do
 			skip=true
 			;;
 
+		-junit)
+			if ! command -v gotestsum 2> /dev/null ; then
+				echo 'E: JUnit output requested but gotestsum not found in PATH. Abort.'
```

Comment:
I was stupidly looking on my PATH for a bit, not the PATH sudo has... could clarify here? But that's a tiny minor nitpick as this is a CI focused change.

Reply:
Would printing out the PATH work?

Reply:
I think that should make it pretty obvious, yes.